src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/changes-since-v3.mdx

```diff
@@ -0,0 +1,230 @@
+---
+title: "Kubernetes integration: what's changed in v3"
```

Comment:
My original idea was to use this page as both things for the moment, and to convert it to an "architectural overview" when we move out of beta.

We could have both now but I think it might be too much effort for the timeline :(